apps/dashboard/src/components/workflow-editor/test-workflow/test-workflow-logs-sidebar.tsx

```diff
@@ -55,33 +57,48 @@ export const TestWorkflowLogsSidebar = ({ transactionId, workflow }: TestWorkflo
     }
   }, [activityId]);
 
-  // Reset refetch when transaction ID changes
+  const handleTransactionIdChange = useCallback((newTransactionId: string) => {
+    setIsTransactionChanging(true);
```

Comment:
I'm not sure that this flag is needed, because the `useFetchActivities` query should be triggered and `areActivitiesPending` should be `true` resulting in the skeleton

Reply:
i was not able to find a reason at this point so i remoed it.